routes/Laundry/get_locations.js

```diff
@@ -0,0 +1,20 @@
+const cheerio = require('cheerio');
+const request = require('request');
+
+module.exports = function getLocations() {
```

Comment:
Can we change this to "Get Location Urls" OR "Get location Ids" to differentiate it from /locations/all?